analyzers/Btagging.py

```diff
@@ -0,0 +1,12 @@
+from heppy.framework.analyzer import Analyzer
+
+class Btagging(Analyzer):
+    
+    def process(self, event):
+        jets = getattr(event, self.cfg_ana.input_objects)
+        bjets = [jet for jet in jets if self.cfg_ana.filter_func(jet)]
```

Comment:
Salut Clement!
no need for two loops, nor to run the filtering twice
do something like

    for jet in jets: 
         if filter_func(jet):
              bjets.append(jet)    
              jet.tag['b'] = True
         else:
              jet.tag['b'] = False